tests/e2e/config.go

```diff
@@ -465,7 +466,7 @@ func CompatibilityTestConfig(providerVersion, consumerVersion string) TestConfig
 				".app_state.provider.params.slash_meter_replenish_fraction = \"1.0\" | " + // This disables slash packet throttling
 				".app_state.provider.params.slash_meter_replenish_period = \"3s\"",
 		}
-	} else if semver.Compare(providerVersion, "v4.0.0") < 0 {
+	} else if semver.Compare(providerVersion, "v4.0.0") <= 0 {
```

Comment:
This was changed from `<` to `<=` because otherwise `v4.0.0` would get the default test configuration that contains the `blocks_per_epoch` parameter and fail with:
```
startChain: Error: failed to validate genesis state: failed to unmarshal provider genesis state: unknown field "blocks_per_epoch" in types.Params
```